dali/server/daldap.cpp

```diff
@@ -194,7 +194,7 @@ class CDaliLdapConnection: implements IDaliLdapConnection, public CInterface
                 user->setAuthenticateStatus(AS_AUTHENTICATED);
             }
             else
-                ERRLOG("LDAP: getPermissions(%s) scope=%s user=%s digital signature support not available",key?key:"NULL",obj?obj:"NULL",username.str());
+                ERRLOG("LDAP: getPermissions(%s) scope=%s user=%s Dali request is signed, but Dali is not configured to verify digital signatures",key?key:"NULL",obj?obj:"NULL",username.str());
```

Comment:
Dali received signed request, however Dali is not configured to verify digital signatures.